ppb/events.py

```diff
@@ -22,16 +22,50 @@ def camel_to_snake(txt):
     return boundaries_finder_2.sub(r'\1_\2', s1).lower()
 
 
+class BadEventHandlerException(TypeError):
+
+    def __init__(self, object_type, method, event_type):
+        article = ['a', 'an'][int(type(event_type).__name__.lower()[0] in "aeiou")]
```

Comment:
Is that an extra `type()` or am I not thinking clearly?

Reply:
It totally is. I rearranged the code and forgot to get this one.